challenge-07/challenge-07.js

```diff
@@ -1,51 +1,72 @@
 /*
 Crie um array com 5 items (tipos variados).
 */
-// ?
+var myArray =[ 'lucas' , { pessoa: 'idade'} , function matriz() {} , 100 , 50 ];
 
 /*
 Crie uma função chamada `addItem`, que irá adicionar itens no array criado.
 A função deverá retornar o array atualizado.
 */
-// ?
+function addItem(add) {
+    return myArray.push(add);
+}
 
 /*
 Adicione um novo array ao array criado no início do desafio, com ao menos 3
 itens de tipos diferentes, mostrando o resultado no console.
 */
-// ?
+addItem( [ 'black' , 'blind' , 'iron']);
 
 /*
 Mostre no console o segundo elemento desse último array, criado acima, com a
 frase:
 "O segundo elemento do segundo array é [ELEMENTO]."
 */
-// ?
-
+console.log('O segundo elemento do segundo array é ' + myArray[5][1]);
+// O segundo elemento do segundo array é blind
 /*
 Mostre no console quantos itens tem o primeiro array criado, com a frase:
 "O primeiro array tem [QUANTIDADE DE ITENS] itens."
 */
-// ?
+console.log('O primeiro array tem ' +  myArray.length+ ' itens.');
+// O primeiro array tem 6 itens.
 
 /*
 Agora mostre no console quantos itens tem o segundo array criado, com a frase:
 "O segundo array tem [QUANTIDADE DE ITENS] itens."
 */
-// ?
+console.log('O segundo array tem ' + myArray[5].length + ' itens.');
+// O segundo array tem 3 itens.
 
 /*
 Utilizando a estrutura de repetição `while`, mostre no console todos os números
 pares entre 10 e 20, inclusive esses 2.
 */
-console.log( 'Números pares entre 10 e 20:' );
-// ?
+
+   function cont(){
```

Comment:
-1
Se você usar funções aqui vai ter problemas. Execute esse arquivo no console que você vai ver o que acontece :)
Use somente as estruturas de repetição. Só use funções quando for pedido pra criar no enunciado, ok? :)